ledger/narwhal/batch-header/src/bytes.rs

```diff
@@ -20,7 +20,7 @@ impl<N: Network> FromBytes for BatchHeader<N> {
         // Read the version.
         let version = u8::read_le(&mut reader)?;
         // Ensure the version is valid.
-        if version != 1 {
+        if version != 1 && version != 2 {
```

Comment:
Should these be constants?

Reply:
I forgot to put a `// TODO: For mainnet` comment on this. This is only a temporary change.